test/server/lib/validation/index.test.js

```diff
@@ -19,71 +18,69 @@ describe.skip('server/lib/validation/applications', () => {
     done();
   });
 
```

Comment:
I think we are missing a test for the `getErrorSignature` method :)

Reply:
added test for getErrorSignature